spring-web/src/main/java/org/springframework/web/context/support/WebApplicationContextUtils.java

```diff
@@ -125,6 +136,63 @@ public static WebApplicationContext getWebApplicationContext(ServletContext sc,
 		return (WebApplicationContext) attr;
 	}
 
+	/**
+	 * Registers a custom {@link WebApplicationContext} for this web app.
```

Comment:
Spring's Javadoc convention is to use an imperative statement for the list line -- in other words, "Register ..." instead of "Registers ...". If you don't follow that rule, you run the risk of getting Juergenized. :wink: